App/ProtoControllers/Conge.php

```diff
@@ -68,7 +68,7 @@ public function getListe()
             $i = true;
             $listeConges = $this->getListeSQL($listId);
             $modifAutorisee = $config->canUserModifieDemande();
-            $affichDateTraitement = $config->canAfficheDateTraitement();
+            $affichDateTraitement = canAfficheDateTraitement(true);
```

Comment:
I don't understand these lines since there is no function named `canAfficheDateTraitement()` in the app. Did you add it in yours and forget to add the file in the PR ?
Plus, please consider removing  `Configuration::canAfficheDateTraitement()`.